### PR TITLE
feat: add price to document title

### PR DESCRIPTION
**Pull request by Rovak**
Adds the current CAKE price to the document title

Screenshot from 2021-02-16 18-17-23

### CODE REVIEW

File: src/DocumentTitle.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import useTitle from 'react-use/lib/useTitle'
+import { usePriceCakeBusd } from './state/hooks'
+
+const DocumentTitle = ({ title, showPrice = true }) => {
```
Comment (hachiojidev) [inline src/DocumentTitle.tsx:5]: Missing types

File: src/DocumentTitle.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  const documentTitle = []
+
+  // Append price if enabled
```
Comment (hachiojidev) [inline src/DocumentTitle.tsx:8]: Try and use comments only if the code needs supplemental explanation. I think this case it is pretty obvious what is happening.

File: src/DocumentTitle.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  const cakePriceUsd = usePriceCakeBusd()
+  if (showPrice) {
+    documentTitle.push(`$${cakePriceUsd.toFixed(3)}`)
+  }
+
+  // Append title
```
Comment (hachiojidev) [inline src/DocumentTitle.tsx:14]: Same here

**Review by hachiojidev — CHANGES_REQUESTED** [review]
Thank you for submitting your PR. In this case I don't think bringing on an extra dependency to change the document title is necessary. If you look at the hook's source you'll see it is pretty simple.

https://github.com/streamich/react-use/blob/master/src/useTitle.ts

**Comment by Chef-Chungus** [comment]
Closed in favor of #494

**Pull request closed without merging**